news/2020-02-03-monthly-beatmapping-contest-february.md

```diff
@@ -10,7 +10,7 @@ date: 2020-02-03 11:00:00 +0000
 
 New month? New mapping contest! Vote on last month's submissions and compete to become the next Elite Mapper!
 
-![monthly beatmapping contest banner](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
```

Comment:
the reason for this is that alt text is pretty much useless for banners, if the image didn't load there would still be room for the banner and the title overlaying it